nntrainer/layers/lstm.cpp

```diff
@@ -228,91 +233,72 @@ void LSTMLayer::forwarding(bool training) {
       acti_func.run_fn(cs, hs);
       hs.multiply_i(ho);
     }
-    // size of h_prev and hs size is same : unit.
-    // size of c_prev and cs is same : unit.
-    h_prev.getBatchSlice(b, 1).copy(hs);
-    c_prev.getBatchSlice(b, 1).copy(cs);
```

Comment:
I wonder if it pass the unittest?

Reply:
Yes, the golden unittest of RNN is in unittest_models which has been activated with this PR.
From what I understood, h_prev was used to cache the values of the current calculation for the next hidden state, and then the next hidden state was initialized with it. 
So, I updated to cache directly in the memory of the next hidden state directly.

Reply:
great, thanks.